cosmwasm/enclaves/shared/cosmwasm-types/v0.10/src/types.rs

```diff
@@ -134,6 +134,11 @@ pub struct TransactionInfo {
     /// using the pair (`env.block.height`, `env.transaction.index`).
     ///
     pub index: u32,
+    /// The hash of the current transaction bytes.
+    /// aka txhash or transaction_id
+    /// hash = sha256(tx_bytes)
+    #[serde(default)]
+    pub hash: String,
```

Comment:
Why are we adding it only to 0.10, and not v1 type?

Reply:
Ahh right right, I know that but I just haven't noticed that v1's `Env` points to the same `TransactionInfo` as v0.10